pyat/at/physics/energy_loss.py

```diff
@@ -68,13 +68,7 @@ def dipole_i2(dipole: Dipole):
     def tracking(ring):
         """Losses from tracking
         """
-        ringtmp = ring.radiation_off(dipole_pass=None,
-                                     quadrupole_pass=None,
-                                     wiggler_pass=None,
-                                     sextupole_pass=None,
-                                     octupole_pass=None,
-                                     copy=True)
-
+        ringtmp = ring.disable_6d(copy=True)
```

Comment:
That's wrong: By default (no argument), disable_6d disables everything, and you get 0 losses! You can use:
```python
ringtmp = ring.disable_6d(RFCavity, copy=True)
```

Reply:
You are right...but I don't understand how this could have worked and went unnoticed...

Reply:
Done